activities/workshops/mapping-risk-and-service-level-agreements/index.md

```diff
@@ -0,0 +1,23 @@
+---
+type: Index
+---
+
+# Mapping risk and service level agreements workshop
+
+## Goal
+
+Have a collaborative working session with new or existing community members to map out, for a given codebase, potential different risks when using the codebase in production, as well as approaches to service level agreements.
+
+## Considerations
+
+Discussions about service level agreements can get very technical quite quickly.
+Try steer the workshop on what the most important risks are, and the process on how they can be mitigated.
```

Comment:
I am not sure exactly how, but it feels like the language of this sentence could be improved.

Reply:
@clausmullie  & @Ainali : What about "Try steer the workshop on what the most important risks are according to the participants. Verify by summarizing and asking participants  about the risks the discussed and how important they are to them. How much do the occur, what is the impact of the risk? Other questions you can ask is what process can be followed to mitigate these risks."

Reply:
Hmm. My hunch was that it would be better with "try to steer" or "try steering" and "towards" instead of "on". But since I am not a native speaker I'll yield that decision to someone else.

Reply:
@ElenaFdR - 'try steer on' seems logical to me, but I don't know why. What would you recommend here?